src/google-maps/README.md

```diff
@@ -19,14 +19,23 @@ To install, run `npm install @angular/google-maps`.
 <!doctype html>
 <head>
   ...
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY">
-  </script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
 </head>
 ```
 
+**Note:**
```

Comment:
There are several classes that we may want to implement in the future that also require adding separate libraries. Should we make this note more general, to apply to several components, while giving more specific instructions about which library to use in the component-specific readme?

Reply:
I think it may be too vague if we're doing it now since there aren't any other components that use this approach. It might be better to do it once we have other components.